tools/mslite

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -e
+pushd $(cd $(dirname ${0})/..; pwd) > /dev/null
+
+bold() {
+    echo "$(tty -s && tput bold)$1$(tty -s && tput sgr0)"
+}
+
+clean() {
+    bold "mindspore clean"
+    rm -rf "./third_party/source/mindspore"
+}
+
+sync() {
+    bold "mindspore sync"
+    mkdir -p "./third_party/source/mindspore/"
+    curl --silent --location --output "./third_party/source/mindspore/model.fbs" "https://github.com/mindspore-ai/mindspore/blob/master/mindspore/lite/schema/model.fbs?raw=true"
```

Comment:
Keep the original path `./third_party/mindspore/mindspore/lite/schema/model.fbs`.